accumulo/src/test/java/org/vertexium/accumulo/AccumuloResource.java

```diff
@@ -0,0 +1,148 @@
+package org.vertexium.accumulo;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.junit.rules.ExternalResource;
+import org.vertexium.GraphConfiguration;
+import org.vertexium.test.GraphTestBase;
+import org.vertexium.util.VertexiumLogger;
+import org.vertexium.util.VertexiumLoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.vertexium.test.GraphTestBase.*;
+
+public class AccumuloResource extends ExternalResource {
```

Comment:
@mwizeman I did play around with them and they work fine but I believe the class has to be static. Not sure we can make it work with abstract GraphTestBase.